libs/shared/sentry/src/lib/next/client.ts

```diff
@@ -34,6 +35,7 @@ export async function initSentryForNextjsClient(
   try {
     Sentry.init({
       ...opts,
+      ignoreErrors: [GENERIC_ERROR_MESSAGE],
```

Comment:
Since this is a shared library, and could be used by other applications with different sets of errors to ignore, the `ignoreErrors` should be provided as an input arg for `initSentryForNextjsClient`

Reply:
I pulled it up into our struct for the sentry configs, and pass it in to both `initSentryForNextjsClient` and `initSentryForNextjsServer`